pygeoapi/provider/mvt.py

```diff
@@ -59,7 +59,11 @@ def __init__(self, provider_def):
             url = urlparse(self.data)
             baseurl = '{}://{}'.format(url.scheme, url.netloc)
             param_type = '?f=mvt'
-            layer = url.path.split('/{z}/{x}/{y}')[0]
+            layer = '/' + self.get_layer()
```

Comment:
@tomkralidis small refactoring to do the layer parsing always through get_layer. In this code bit we need to add a '/' at the beginning of the url